content/en/blog/2021/announcing-notation-alpha1.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Announcing notation alpha 1
+spelling: cSpell:ignore 
```

Comment:
we can probably remove the spelling line if you've not got any words in the copy that are triggering the spell checker that shouldn't be.